### ISSUE TITLE
refresh cscope-db reports Error: Unable to refresh cscope database

### ISSUE THREAD

**Issue by davy79** [amitab/atom-cscope#5]
Unfortunately the refresh-db command reports Error: Unable to refresh cscope database.

Running ubuntu 14.04 with cscope running from the command line.
Atom 1.2.4

Is there any other debug info I can provide?

**Comment by amitab** [comment]
If this fails, some object would have been printed in the console. 
Example: 
`Object {success: false, message: "At /home/amitabh/src/mysql-router: find: paths mus…arch|stat|rates|opt|exec] [path...] [expression]↵"}`
The attribute: `message` would contain the reason why it failed. Can you paste that here?

**Comment by davy79** [comment]
hmmm, I got this...

```
ReferenceError: fs is not defined
    at /home/jda/.atom/packages/atom-cscope/lib/cscope.coffee:20:7
    at Object.module.exports.CscopeCommands.writeToFile (/home/jda/.atom/packages/atom-cscope/lib/cscope.coffee:19:16)
    at /home/jda/.atom/packages/atom-cscope/lib/cscope.coffee:31:17
    at process._tickCallback (node.js:368:9)
```

**Comment by amitab** [comment]
Resolved in 42ffebfb1f99162d492cc02959f61d70dadcb0b7
Please check the new release and let me know if the same problem persists. Honestly I have no idea why it was working for me even though the variable `fs` was not declared.

**Comment by davy79** [comment]
Verified!

Everything works fine now!

It's a common problem that something works on our own machine :smile: 
most common response from the dev after receiving a bug report: "works for me"

At work we always test on a clean virtual machine before release to avoid such common issue.

Thanks!
